jetty-openid/src/main/config/modules/openid.mod

```diff
@@ -18,8 +18,14 @@ etc/openid-baseloginservice.xml
 etc/jetty-openid.xml
 
 [ini-template]
-## The OpenID Identity Provider
-# jetty.openid.openIdProvider=https://accounts.google.com/
+## The OpenID Identity Provider's issuer ID (the entire URL *before* ".well-known/openid-configuration")
+# jetty.openid.openIdProvider=https://id.example.com/~
+
+## The OpenID Identity Provider's authorization endpoint (optional if the metadata of the OP is accessible)
+# jetty.openid.openIdProvider=https://id.example.com/authorization
```

Comment:
Forgot to set the correct property names here.
`jetty.openid.authorizationEndpoint` and
`jetty.openid.tokenEndpoint`